firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -470,12 +489,17 @@ public void beginRealtimeHttpStream() {
       if (connectionFailed || responseCode == HttpURLConnection.HTTP_OK) {
         retryHttpConnectionWhenBackoffEnds();
       } else {
+        String errorMessage =
+            String.format(
+                "Unable to connect to the server. Try again in a few minutes. Http Status code: %d",
+                responseCode);
+        if (responseCode == 403) {
```

Comment:
What if we get a 403 without a message we expect? We should still return some error message in the exception.